src/geometries/TorusGeometry.js

```diff
@@ -4,7 +4,7 @@ import { Vector3 } from '../math/Vector3.js';
 
 class TorusGeometry extends BufferGeometry {
 
-	constructor( radius = 1, tube = 0.4, radialSegments = 8, tubularSegments = 6, arc = Math.PI * 2 ) {
+	constructor( radius = 1, tube = 0.4, radialSegments = 12, tubularSegments = 48, arc = Math.PI * 2 ) {
```

Comment:
It's necessary to update the documentation pages so they reflect the new default values. E.g.:

https://github.com/mrdoob/three.js/blob/6087f9b1fdbd81a3a95123ca124fef9dc9aaa5b6/docs/api/en/geometries/TorusGeometry.html#L49

Reply:
I wasn't sure whether to update the documentation. I'll get to it later today. It might take a bit (Curse not having actual git access), but I will get it done. Just the English documentation, or are the other languages easy enough to change, such that you want me to change those as well?

Reply:
Do as you like. Since we talk about numbers, it should be easy and quick to change the defaults everywhere. If you don't do it, I'll take care of it anyway 😉 .

Reply:
I updated all the languages with geometries folders. As soon as the checks go through, it should be fine.